cassandra/README.md

```diff
@@ -0,0 +1,263 @@
+Command:
+  Type: read
+  Count: -1
+  Duration: 60 SECONDS
+  No Warmup: false
+  Consistency Level: LOCAL_ONE
+  Target Uncertainty: not applicable
+  Key Size (bytes): 10
+  Counter Increment Distibution: add=fixed(1)
+Rate:
+  Auto: false
+  Thread Count: 100
+  OpsPer Sec: 0
+Population:
+  Distribution: Gaussian:  min=1,max=1000000,mean=500000.500000,stdev=166666.500000
+  Order: ARBITRARY
+  Wrap: false
+...
+```
+
+**9. You can open a shell to the database, if you'd like to poke around.**
+```bash
+$ ./csstress_script.sh muddled-molly shell
+/usr/bin/kubectl run --namespace default muddled-molly-cassandra-stress-shell --restart=Never --rm --tty -i --image cassandra --command -- cqlsh muddled-molly-cassandra
+If you don't see a command prompt, try pressing enter.
+cqlsh> select * from system_schema.keyspaces;
+
+ keyspace_name      | durable_writes | replication
+--------------------+----------------+-------------------------------------------------------------------------------------
+        system_auth |           True | {'class': 'org.apache.cassandra.locator.SimpleStrategy', 'replication_factor': '1'}
+      system_schema |           True |                             {'class': 'org.apache.cassandra.locator.LocalStrategy'}
+
+```
+
+**10. In another shell, watch the state of the containers, so that we can see what happens when we kill it.**
+```bash
+$ kubectl get pods -o wide | grep 'muddled-molly'
+muddled-molly-cassandra-0                1/1       Running   0          38m       10.244.3.22     k8s-slave3   <none>
+muddled-molly-cassandra-1                1/1       Running   0          35m       10.244.1.32     k8s-slave1   <none>
+muddled-molly-cassandra-2                1/1       Running   0          33m       10.244.2.40     k8s-slave2   <none>
+```
+
+**11. Now, execute a couple of commands to kill the container, and ensure that it moves to another host.**
+```bash
+$ ./csstress_script.sh muddled-molly kill-and-move
+/usr/bin/kubectl taint node k8s-slave3 key=value:NoSchedule && /usr/bin/kubectl delete pod muddled-molly-cassandra-0
+node/k8s-slave3 tainted
+pod "muddled-molly-cassandra-0" deleted
+```
+
+**12. In our other shell, we'll see that container be terminated, and recreated on another host.**
+```bash
+$ kubectl get pods -o wide | grep 'muddled-molly'
+muddled-molly-cassandra-0                1/1       Running   0          1h        10.244.3.22     k8s-slave3   <none>
+muddled-molly-cassandra-0                0/1       Terminating   0          1h        10.244.3.22     k8s-slave3   <none>
+muddled-molly-cassandra-1                1/1       Running       0          1h        10.244.1.32     k8s-slave1   <none>
+muddled-molly-cassandra-2                1/1       Running       0          58m       10.244.2.40     k8s-slave2   <none>
+muddled-molly-cassandra-0                0/1       ContainerCreating   0          7s        <none>          k8s-slave4   <none>
+```
+
```

Comment:
Are there Cassandra recovery steps that _should_ be executed prior to a `kill-and-move`? I seem to recall a `recover` command or something.

We can also note in the README here that the data moved _with the pod_, so didn't need to be completely rebuilt from the other partitions.